test/execution_test.ts

```diff
@@ -776,6 +776,67 @@ describe('Execution', () => {
           });
         });
 
+        it('context includes sync for validateParameters on sync formula', async () => {
```

Comment:
Tests are failing locally so looking into that, but wanted to get the shift over to metadata formula reviewed